### DISCUSSION TITLE
Ability to define an Update sensor

### DISCUSSION THREAD

**Discussion by LordMike** [zachowj/node-red-contrib-home-assistant-websocket#1396] [Feature Request]
Awesome integration - I'm using this extensively.

Lately, I've made a flow that will update my Esphome devices when I need to. It's a lengthy process of compiling on low powered hardware, so I made a flow which updates one device at a time. Once the esphome devices are ready to update, they show up in HA in settings (as updateable entities), and then I can move in to my Node-red flow and trigger it.

What I'd like to be able to do, is to add an update sensor much like I can already add text/button/sensor entities from Node-red. Then I could expose the "update ready" status on this, and see _this_ combined update entity in settings and click it right there. It would also be able to show a progress bar which mimiced the 20 something esphome devices I have.

That'd be cool :)

**Comment by zachowj** [comment]
In the meantime, you could try using the [mqtt update](https://www.home-assistant.io/integrations/update.mqtt/).

**Reply by LordMike** [reply]
Ooh.. That could be an opportunity.
